scripts/localstack

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
can we remove this since we have docker-compose?

Reply:
I think there’s an issue with creating databases for MS SQL through docker compose which is why I had to do this convoluted way 😕, can double check on this as well

Reply:
removed